packages/platform-sdk/src/helpers.ts

```diff
@@ -76,9 +76,12 @@ export const randomHost = (hosts: NetworkHost[], type: NetworkHostType): Network
 export const filterHostsFromConfig = (config: Config, type: NetworkHostType): NetworkHost[] =>
 	filterHosts(config.get<NetworkHost[]>("network.hosts"), type);
 
-export const randomHostFromConfig = (config: Config, type: NetworkHostType): NetworkHost =>
+export const randomNetworkHostFromConfig = (config: Config, type: NetworkHostType = "full"): NetworkHost =>
 	randomHost(config.get<NetworkHost[]>("network.hosts"), type);
 
+export const randomHostFromConfig = (config: Config, type: NetworkHostType = "full"): string =>
```

Comment:
I would even rename this to `randomHost`. The fact that it comes from config is always clear from the first argument at call sites